src/NumFieldOrd/NfOrd/Ideal/Ideal.jl

```diff
@@ -68,6 +68,11 @@ parent_type(::Type{AbsNumFieldOrderIdeal{AbsSimpleNumField, AbsSimpleNumFieldEle
 
 ==(x::AbsNumFieldOrderIdealSet, y::AbsNumFieldOrderIdealSet) = x.order === y.order
 
+function Base.hash(x::AbsNumFieldOrderIdealSet, h::UInt)
+  h = hash(x.order, h)
+  return xor(h, 0x5d49d4dc68dec3e2 % UInt)
```

Comment:
As for `AlgAssAbsOrdIdlSet`, this could use `objectid(x.order)` for a stronger hash